webpack.config.sample.js

```diff
@@ -43,78 +45,79 @@ let localOwaFolder;
 
 let devtool;
 
-var getConfig = function() {
-    var config;
-
-    try {
-        // look for config file
-        config = require('./config.json');
-    } catch (err) {
-        // create file with defaults if not found
-        config = {
-            'LOCAL_OWA_FOLDER': '/Users/redd/openmrs/openmrs-server/owa/',
-            'APP_ENTRY_POINT': 'http://localhost:8080/openmrs/owa/cohortbuilder/index.html'
-        };
+const getConfig = function () {
+		let config;
 
-        fs.writeFile('config.json', JSON.stringify(config));
+		try {
+			// look for config file
+			config = require('./config.json');
+		} catch (err) {
+			// create file with defaults if not found
+			config = {
+				'LOCAL_OWA_FOLDER': '/home/sims01/openmrs/openmrs-server/owa/',
```

Comment:
No, it won't, that's a sample webpack config file that new developers can use as a template for their own local webpack config file, and they have to specify the accurate path to their own local owa directory.